frontend/pages/medical-recovery-centre/index.js

```diff
@@ -89,14 +90,15 @@ const style = (
     }
     .card-margin {
       margin: 0 !important;
-    }
+    }import SocialShare from './../../components/SocialShare';
```

Comment:
Should the import be here?

Reply:
no it shouldn't be ... i think my auto import is broken xDD